llpc/context/llpcCompiler.cpp

```diff
@@ -211,10 +211,28 @@ static void fatalErrorHandler(void *userData, const std::string &reason, bool ge
 #endif
 }
 
+static void InlineAsmDiagHandler(const SMDiagnostic &SMD, void *Context, unsigned LocCookie) {
```

Comment:
Please add function header comments and pay attention to the coding style
SMD -> sourceMgrDiag
Context -> context
LocCookie -> locCookie